src/HasPrefixCollection.php

```
@@ -36,30 +27,102 @@ trait HasPrefixCollection
+     * ```php
+     * $id = $component->getPrefixAttribute('id', 'default-id');
+     * ```
+     *
+     * @param string|UnitEnum $key Attribute name.
+     * @param mixed $default Default value when the attribute is missing.
+     *
+     * @return mixed Attribute value or default.
+     */
+    public function getPrefixAttribute(string|UnitEnum $key, mixed $default = null): mixed
+    {
+        $normalizedKey = Enum::normalizeValue($key);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\Enum' in method 'getPrefixAttribute'. (undefined)

(StaticAccess)